bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/AbstractHomeConnectThingHandler.java

```diff
@@ -469,6 +470,21 @@ protected boolean isThingOffline() {
         }
     }
 
+    /**
+     * Get thing linked channel by given channel id.
+     *
+     * @param channelId channel id
+     * @return channel if linked
+     */
+    protected Optional<Channel> getThingLinkedChannel(String channelId) {
```

Comment:
I would prefer `getLinkedChannel` but up to you.

Reply:
I agree with you.